src/plugins/intel_gpu/src/graph/graph_optimizer/handle_reshape.cpp

```diff
@@ -181,6 +181,10 @@ void handle_reshape::run(program& p) {
                                        *reorder_reshape_node,
                                        0,
                                        reshape_input_node.get_dependencies().empty());
+
+                    if (reorder_reshape_node->get_primitive()->input_size() == 2) {
```

Comment:
Wouldn't it be better to move this node modification to [node creation part](https://github.com/openvinotoolkit/openvino/blob/22bb3af7df965dc85f7b4d506f9649ecedaff775/src/plugins/intel_gpu/src/graph/graph_optimizer/handle_reshape.cpp#L159) rather than apply it in reorders adding stage?

Reply:
updated